docs/content/documentation/getting-started/overview.md

```diff
@@ -0,0 +1,202 @@
+
+```bash
+├── config.toml
+├── content
+├── sass
+├── static
+├── templates
+└── themes
+```
+
+Let's start the zola development server with:
+
+```bash
+$ zola serve
+Building site...
+-> Creating 0 pages (0 orphan), 0 sections, and processing 0 images
+```
+
+> This command must be run in the base Zola directory, which contains `config.toml`.
+
+If you point your web browser to [127.0.0.1:1111](http://127.0.0.1:1111), you should see a "Welcome to Zola" message.
+
+### Home Page
+
+Let's make a home page. To do this, let's first create a `base.html` file inside the `templates` directory. This step will make more sense as we move through this overview. We'll be using the CSS framework [Bulma](https://bulma.io/).
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>MyBlog</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.8.0/css/bulma.min.css">
+</head>
+
+<body>
+  <section class="section">
+    <div class="container">
+      {% block content %} {% endblock %}
+    </div>
+  </section>
+</body>
+
+</html>
+```  
+
+Now, let's create an `index.html` file inside the `templates` directory.
+
+```html
+{% extends "base.html" %}
+
+{% block content %}
+<h1 class="title">
+  This is my blog made with Zola.
+</h1>
+{% endblock content %}
+```  
+
+This tells Zola that `index.html` extends our `base.html` file and replaces the block called "content" with the text between the `{% block content %}` and `{% endblock content %}` tags.
+
+### Content Directory
+
+Now let's add some content. We'll start by making a `blog` subdirectory in the `content` directory and creating an `_index.md` file inside it. This file tells Zola that `blog` is a [section](@/documentation/content/section.md), which is how content is categorized in Zola.
+
+```bash
+├── content
+│   └── blog
+│       └── _index.md
+```
+
+In the `_index.md` file, we'll set the following variables in [TOML](https://github.com/toml-lang/toml) format:
+
+```md
++++
+title = "List of blog posts"
+sort_by = "date"
+template = "blog.html"
+page_template = "blog-page.html"
++++
+```
+
+> Note that although no variables are mandatory, the opening and closing `+++` are required.
+
+*sort_by = "date"* tells Zola to use the date to order our section pages (more on pages below). *template = "blog.html"* tells Zola to use `blog.html` in the `templates` directory as the template for listing the Markdown files in this section. *page_template = "blog-page.html"* tells Zola to use `blog-page.html` in the `templates` directory as the template for individual Markdown files. For a full list of section variables, please see the [section](@/documentation/content/section.md) documentation. We will use *title = "List of blog posts"* in a template (see below).
+
+### Templates
+
+Let's now create some templates. In the `templates` directory, create a `blog.html` file with the following contents:
```

Comment:
We've already created the homepage! Maybe "some more templates"?

Reply:
Oops. Yes, "some more".